versioned_docs/version-8.5/apis-tools/tasklist-api-rest/migrate-to-zeebe-user-tasks.md

```diff
@@ -370,12 +370,6 @@ type:"link",
 href:"/docs/next/apis-tools/tasklist-api-rest/tasklist-api-rest-overview/",
 label: "Tasklist API (REST)",
 docId:"apis-tools/tasklist-api-rest/tasklist-api-rest-overview"
-},
-{
-type:"link",
-href:"/docs/next/apis-tools/zeebe-api-rest/zeebe-api-rest-overview/",
-label: "Zeebe API (REST)",
-docId:"apis-tools/zeebe-api-rest/zeebe-api-rest-overview"
```

Comment:
@christinaausley, should we really just remove this from here? Should we rather adjust this to point to the C8 REST API?

Reply:
This versioned change snuck in. Will revert.